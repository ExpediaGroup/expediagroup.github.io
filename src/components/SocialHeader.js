import React from 'react';
import styles from './SocialHeader.module.css';

function SocialHeader({title}) {
    return (
        <div className={styles.socialHeader}>
            <h1>{title}</h1>
        </div>
    );
}

export default SocialHeader;
