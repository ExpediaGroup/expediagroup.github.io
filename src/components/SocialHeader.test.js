import React from 'react';
import renderer from 'react-test-renderer';
import SocialHeader from './SocialHeader';

it('renders correctly', () => {
    const tree = renderer
        .create(<SocialHeader title="foo"/>)
        .toJSON();
    expect(tree).toMatchSnapshot();
});
