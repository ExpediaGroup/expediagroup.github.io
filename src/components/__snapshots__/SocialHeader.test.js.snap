// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="socialHeader"
>
  <h1>
    test title
  </h1>
</div>
`;
